Makefile.work

```diff
@@ -183,6 +183,7 @@ SONIC_BUILD_INSTRUCTION :=  make \
                            SONIC_ENABLE_SYSTEM_TELEMETRY=$(ENABLE_SYSTEM_TELEMETRY) \
                            SONIC_ENABLE_RESTAPI=$(ENABLE_RESTAPI) \
                            EXTRA_JESSIE_TARGETS=$(EXTRA_JESSIE_TARGETS) \
+                           TELEMETRY_READWRITE=$(TELEMETRY_READWRITE) \
```

Comment:
Please update the 'Supported parameters:' list at the top.
Also update rules/config, with the added lines under comments, as it is not used by default.

Reply:
Okay, dones.